docs/community/gsoc.md

```diff
@@ -0,0 +1,33 @@
+# `envd` Summer of Code 2023
+
+Google Summer of Code ([GSoC](https://summerofcode.withgoogle.com/)) is a global, online program focused on engaging new contributors in open source software development. During the 12+ weeks of programming under the guidance of mentors, you will gain hands-on experience in open source software projects.
+
+`envd` is a CLI tool that helps people create the container-based development and deployment environment for AI/ML. We use the following technologies:
+
+- [BuildKit](https://github.com/moby/buildkit)
+- [Starlark](https://github.com/bazelbuild/starlark)
+- [ContainerSSH](https://github.com/containerssh/containerssh)
+
+And we're intending to apply to be part of GSoC 2023!
+
+## Get started
+
+Check out our [contributing](/community/contributing) and [development](/developers/development) guides to learn more about how to get started with `envd`.
+
+## Prerequisites
+
+Make sure you meet the [GSoC eligibility requirements](https://developers.google.com/open-source/gsoc/faq#what_are_the_eligibility_requirements_for_participation).
+
+If possible, please make at least one pull request to any projects under [TensorChord](https://github.com/tensorchord/).
+
+We would prefer the applicants with container or machine learning experience.
+
+## Application instructions
+
+- Start with a clear and concise description of the problem you want to solve or the feature you want to add.
+- If you have any previous experience working on similar projects, include examples or links to relevant work.
+- Is there anything that you'll be studying or working on while working with us?
```

Comment:
Use declarative sentence instead of question here?

Reply:
Changed to another declarative sentence. PTAL